src/core/namespace.c

```diff
@@ -295,7 +295,7 @@ static int append_empty_dir_mounts(MountEntry **p, char **strv) {
                         .mode = EMPTY_DIR,
                         .ignore = false,
                         .read_only = true,
-                        .options_const = "mode=755",
+                        .options_const = "mode=755,size=10%",
```

Comment:
this should probably be size=1, because it is really supposed to be empty and read-only, and never should contain any data whatsoever